core/src/style/style.h

```diff
@@ -142,7 +142,7 @@ using StyleUniform = std::pair< std::string, UniformValue >;
     virtual void onEndBuildTile(Tile& _tile) const;
 
     /* Perform any setup needed before drawing each frame */
-    virtual void onBeginDrawFrame(const View& _view, Scene& _scene);
+    virtual void onBeginDrawFrame(const View& _view, Scene& _scene, int _textureUnit = 0);
```

Comment:
Is `_textureUnit` the next available texture unit?  The last-bound texture unit? A comment might be good.